homeassistant/components/v2c/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Config flow for V2C integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pytrydan import Trydan
+from pytrydan.exceptions import TrydanError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("host"): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
This function is very small, I would suggest just using it inline and catching the TrydanError directly